beanstalk_worker/worker_start_scan.py

```diff
@@ -433,5 +433,11 @@ def process_output(self, json_data):
             msg="Job moved from scan phase, id: %s" % job_id
             )
         job.delete()
+    except beanstalkc.CommandFailed as e:
+        logger.log(
+            level=logging.ERROR,
+            msg="Failed to put scanner data on beanstalkd tube; deleting job"
+        )
+        job.delete()
```

Comment:
deleting the job is needed in either case, whether scanner ran successfully or there was an exception, deleting the job in `finally` clause would make more sense. With logs UI feature, this is implemented in the https://github.com/CentOS/container-pipeline-service/pull/125 with additional code.